cmd/meroxa/turbine/python/upgrade.go

```diff
@@ -4,41 +4,41 @@ import (
 	"fmt"
 	"os/exec"
 
+	utils "github.com/meroxa/cli/cmd/meroxa/turbine"
 	"github.com/meroxa/cli/log"
 )
 
 const turbinePYVersion = "1.5.3"
 
 // Upgrade fetches the latest Meroxa dependencies.
 func (t *turbinePyCLI) Upgrade(vendor bool) error {
-	cmd := exec.Command("grep", "turbine-py==", "requirements.txt")
```

Comment:
We're choosing to run `pip upgrade` specifically on our libraries regardless of what is installed. Nothing would change if they are on the most up to date version.